### PR TITLE
Document API changes supporting new Droplet backup policies

### PR DESCRIPTION
**Pull request by andrewsomething**

### CODE REVIEW

**Comment by andrewsomething** [comment]
Will need a re-base on main once https://github.com/digitalocean/openapi/pull/912 lands.

File: specification/resources/droplets/models/droplet_actions.yml [hunk 1]
```diff
@@ -25,6 +25,42 @@ droplet_action:
...
       description: The type of action to initiate for the Droplet.
 
+droplet_action_enable_backups:
+  allOf:
+  - $ref: '#/droplet_action'
+  - type: object
+    properties:
+      backup_policy:
+        allOf:
+        - $ref: 'droplet_backup_policy.yml'
```
Comment (loosla) [inline specification/resources/droplets/models/droplet_actions.yml:35]: :+1: Looks great.
Just curious if there is a way to clarify how the default backup policy looks like?

**Review by loosla — APPROVED** [review]
:+1:

File: specification/resources/droplets/models/droplet_actions.yml (hunk 1 above)
Comment (andrewsomething) [reply specification/resources/droplets/models/droplet_actions.yml:35]: That's a good question! We should fast follow with some more details. @house-lee is the default weekly or daily?

**Pull request merged**